lib/schemas.js

```diff
@@ -27,11 +27,12 @@ const defaults = {
 }
 
 const optionsSchema = Joi.object({
-  userConfigPath: Joi.string(),
-  baseDir: Joi.string(),
-  onConfig: Joi.func(),
+  baseConfig: Joi.string(),
+  baseDir: Joi.string().allow(null),
   environment: Joi.object().default(defaults.environment),
-  shortstopHandlers: Joi.object().default(defaults.shortstopHandlers)
+  onConfig: Joi.func(),
+  shortstopHandlers: Joi.object().default(defaults.shortstopHandlers),
+  userConfigPath: Joi.array().items(Joi.string()).single()
```

Comment:
Is `baseConfig` necessary anymore since `userConfigPath` exists?

Reply:
nope!